resources/assets/js/processes/components/ProcessesListing.vue

```diff
@@ -85,10 +85,36 @@
         },
 
         methods: {
+            activateBtnCssClass (data) {
+                var showPowerOn = (data.status === 'INACTIVE') ? true : false;
+                var showPowerOff = (data.status === 'ACTIVE') ? true : false;
+                return {
+                    'fa-toggle-off': showPowerOff,
+                    'fa-toggle-on': showPowerOn
+                };
+            },
+            activateBtnTitle (data) {
+                return (data.status === 'ACTIVE') ? 'Deactivate' : 'Activate'
+            },
             onAction (actionType, data, index) {
                 if (actionType === 'edit-item') {
                     window.open('/designer/' + data.uid);
                 }
+
+                if (actionType === 'toggle-status') {
+                    this.loading = true;
+                    ProcessMaker.apiClient
+                        .put('/processes/' + data.uid, {
+                            status: (data.status === 'ACTIVE') ? 'INACTIVE' : 'ACTIVE'
+                        })
+                        .then(response => {
+                            this.loading = false;
+                            document.location.reload();
+                        })
+                        .catch(error => {
+                            // Undefined behavior currently, show modal?
```

Comment:
you could use it 

ProcessMaker.alert(message, type)

to inform you that the state could not be changed.

Reply:
We're already catching the error. We should not be including empty methods.

Reply:
With #399 the api errors will be shown by default with an alert.

Reply:
The catch section of the api calls was removed